modular_skyrat/modules/inflatables/code/inflatable.dm

```diff
@@ -124,118 +85,127 @@
 
 	if(usr.stat || usr.can_interact())
 		return
-	deflate()
+	deflate(FALSE)
 
-/obj/item/inflatable/door
-	name = "inflatable door"
-	desc = "A folded membrane which rapidly expands into a simple door on activation."
-	icon = 'modular_skyrat/modules/inflatables/icons/inflatable.dmi'
-	icon_state = "folded_door"
-	structuretype = /obj/structure/inflatable/door
 
-/obj/structure/inflatable/door //based on mineral door code
+/obj/structure/inflatable/door
 	name = "inflatable door"
-	density = TRUE
-	anchored = TRUE
 	can_atmos_pass = ATMOS_PASS_DENSITY
 	icon = 'modular_skyrat/modules/inflatables/icons/inflatable.dmi'
 	icon_state = "door_closed"
-	torntype = /obj/item/inflatable/torn/door
-	itemtype = /obj/item/inflatable/door
-	var/state = FALSE //closed, 1 == open
-	var/isSwitchingStates = FALSE
-
-/obj/structure/inflatable/door/attack_hand(mob/user)
-	return TryToSwitchState(user)
-/*
-/obj/structure/inflatable/door/CanPass(atom/movable/mover, turf/target, height=0, air_group=0)
+	base_icon_state = "door"
+	torn_type = /obj/item/inflatable/door/torn
+	deflated_type = /obj/item/inflatable/door
+	/// Are we open(FALSE), or are we closed(TRUE)?
+	var/door_state = TRUE
```

Comment:
That should be two new defines, being `INFLATABLE_DOOR_OPENED` (0) and `INFLATABLE_DOOR_CLOSED` (1), that'd be less confusing.